scripts/styleguide.sh

```diff
@@ -10,7 +10,9 @@ styleguide \
   --server \
   --disableEncapsulation \
   --title 'bitstyles' \
+  --extraHead '<script src="/assets/javascripts/ofi.js"></script>' \
   --extraHead '<script src="/assets/javascripts/a11y-dialog.js"></script>' \
   --extraHead '<script src="/assets/javascripts/styleguide-init.js"></script>' \
   --extraHead '<script src="/assets/javascripts/modal.js"></script>' \
   --extraHead '<script src="/assets/javascripts/expander.js"></script>' \
+  --extraHead '<script src="/assets/javascripts/object-fit-polyfill.js"></script>' \
```

Comment:
@RRorg The styleguide server doesn't have access to the `node_modules` folder, this was supposed to be checked in (like the a11y-dialog script) — I missed that it wasn’t, good catch! Updated.